ui/src/app/pages/access/compliance-training-module-card-title.spec.tsx

```diff
@@ -132,29 +159,32 @@ describe(ComplianceTrainingModuleCardTitle.name, () => {
       }
     );
 
-    await expectHelpTextToExist();
+    expectHelpTextToExist();
+    expectMaintenanceTextToNotExist();
   });
 
-  it('does not show help text if module is bypassed', async () => {
+  it('shows maintenance text if compliance training is complete and module is expiring and training is in a maintenance window', async () => {
     setup(
       {
         ...createProps(),
         profile: createProfileWithComplianceTraining(
-          null,
-          null,
-          nowPlusDays(-1)
+          nowPlusDays(-1),
+          nowPlusDays(29),
+          null
         ),
       },
       {
         ...defaultServerConfig,
+        blockComplianceTraining: true,
         complianceTrainingRenewalLookback: 30,
       }
     );
 
-    await expectHelpTextToNotExist();
+    expectMaintenanceTextToExist();
+    expectHelpTextToNotExist();
   });
 
-  it('does not show help text if absorb is not used', async () => {
```

Comment:
Duplicate test "does not show help text if module is bypassed" and "does not show help text if module is bypassed"